spec/requests/api_spec.rb

```diff
@@ -55,6 +55,11 @@
 
     describe "GET KGS Username" do
       it "returns a 200 status code for a username that does exist" do
+        skip <<-EOS.squish.freeze
+          Intermittently failing with "OpenURI::HTTPError: 503". We should
+          rewrite this test to avoid actual network operations, by using
+          rspec-mocks, or more advanced techniques like webmock or vcr.
+        EOS
```

Comment:
Either before or after the e-Go Congress, I want to take you up on your offer to pair on this. I'd really like to get webmock working for this.